### DISCUSSION TITLE
How to make "tel:" (phone call) links in Markdown?

### DISCUSSION THREAD

**Discussion by lionkmp** [gsantner/markor#1629] [Q&A] [answered]
I notice they should work as explained in this discussion: https://github.com/gsantner/markor/issues/749

I have tel links exactly like that. `[+123456](tel:+123456)`

They are not displayed as a link but as plain text. I have many e-mail links too in the same file, they work OK.

Using version: 2.8.6 (130) from F-Droid.

Thanks in advance.

**Comment by gsantner** [comment, marked as answer]
Works fine as it is. And directly starts phone application on click with number prefilled.

you just need to escape the plus in the title, if it starts with it.  
`[\+123456](tel:+123456)`

**Comment by lionkmp** [comment]
Thank you, I'll do that.

Please note that this is not obvious, because I open these MD files with "Okular" on Linux, and also convert them to PDF with pandoc, both handled my links without needing the escape the leading + sign. 

Keep up the good work! Markor is excellent! :-)

**Comment by gsantner** [comment]
you need to escape the plus in the title. Works fine. And directly starts phone application on click with number prefilled.


`[\+123456](tel:+123456)`

Am 8. März 2022 21:50:01 MEZ schrieb lionkmp ***@***.***>:
